components/accelerator/nvidia/query/nvml/nvml.go

```diff
@@ -385,7 +387,7 @@ func (inst *instance) Start() error {
 		}
 	}
 
-	if inst.xidErrorSupported {
+	if inst.xidErrorSupported && inst.xidWatchWithNVML {
 		go inst.pollXidEvents()
```

Comment:
what is this? can we delete this func all together?

Reply:
removed as well